### ISSUE TITLE
Add level and Parent fields for Categories query

### ISSUE THREAD

**Issue by jan-thoma** [markhuot/craftql#149]
Add level and Parent fields for Categories query. Especially helpful when creating category navigations

**Comment by markhuot** [comment]
This should already be possible. Check out,

```graphql
{
  categories(level:1, ancestorOf:123) {
    #...
  }
}
```

In the above `level` is the depth and `ancestor` takes the entry ID of the parent you're looking for.

**Comment by markhuot** [comment]
Ah, I see! These new fields are implemented on `dev-master`. I'll leave this ticket open until it gets a stable release.

```graphql
{
  category {
    id
    title
    parent {
      id
    }
    children {
      id
    }
    next { id }
    nextSibling { id }
    prev { id }
    prevSibling { id }
  }
}
```

**Comment by jan-thoma** [comment]
I will give it try, i switched to dev-master the last day, querys on the dev master took up to 8 Seconds on our server. Is there something to consider when switching the branch?

**Comment by markhuot** [comment]
Nope, nothing performance wise should have changed. between `1.1.0` and `dev-master`. You will want to make sure `devMode` is `false` though otherwise GraphQL will spend a bunch of time analyzing your schema on each request. That's been there for a while though so it shouldn't vary by branch.

**Comment by jan-thoma** [comment]
will check that and open a new issue if the problem resides.Am 07.09.2018 20:50 schrieb Mark Huot <notifications@github.com>:Nope, nothing performance wise should have changed. between 1.1.0 and dev-master. You will want to make sure devMode is false though otherwise GraphQL will spend a bunch of time analyzing your schema on each request. That's been there for a while though so it shouldn't vary by branch.

**Comment by mrjacobfisher** [comment]
I'm interested in whether this is planned for a release. It would really simplify some things I'm currently working on over the coming weeks and I'd be curious to know about progress.

**Comment by mashema** [comment]
Same here, I'm also waiting, do you have a release date?

**Comment by markhuot** [comment]
Closing this issue. The fix is included in version 1.2.1.
